app/src/main/java/com/github/se/cyrcle/ui/parkingDetails/ParkingDetailsScreen.kt

```diff
@@ -88,6 +88,11 @@ fun ParkingDetailsScreen(
   val showDialog = remember { mutableStateOf(false) }
   val imagesUrls by parkingViewModel.selectedParkingImagesUrls.collectAsState()
   val showDialogImage = remember { mutableStateOf<String?>(null) }
+  if (selectedParking.owner != "Unknown Owner" && selectedParking.owner != null) {
```

Comment:
It's possible. Since there are only two distinct cases here I don't think creating the function itself would be too worth it for now, I'll keep it in mind if this comes back up